test/opexecuter.cpp

```diff
@@ -297,10 +305,13 @@ void OpExecuter::ExecuteOnNGraph(vector<Tensor>& ngraph_outputs,
   shared_ptr<ng::Function> ng_function;
   ASSERT_EQ(Status::OK(),
             Builder::TranslateGraph(input_shapes, static_input_map, &graph,
-                                    ng_function));
+                                    ng_function))
+      << "Failed to TranslateGraph" << std::endl;
 
   // ng function should get same number of outputs
-  ASSERT_EQ(expected_output_datatypes_.size(), ng_function->get_output_size());
+  ASSERT_EQ(expected_output_datatypes_.size(), ng_function->get_output_size())
+      << "Number of outputs of requested outputs and ngraph function outputs "
+         "do not match";
```

Comment:
Similar to @shresthamalik 's comment, I see at some places you have `std::endl` in the end, and some don't. Maybe change them to be consistent.